.changeset/slimy-starfishes-prove.md

```diff
@@ -0,0 +1,5 @@
+---
+'rollup-plugin-workbox': major
```

Comment:
This should be a major version right?

This package also doesn't seem to have any tests so I'm unsure how to verify if  this is breaking.

Reply:
esbuild doesnt follow semver so it does breaking changes on minor/patch versions, so technically this would be a breaking change. I think it's probably fine to do it in a minor though.